packages/cli/src/index.ts

```diff
@@ -63,68 +67,100 @@ const { input, flags } = meow(
 const cwd = process.cwd();
 
 (async () => {
+  const workspaces = await getWorkspaces({
+    cwd,
+    tools: ["yarn", "bolt", "root"]
+  });
+
+  if (!workspaces) {
+    throw new Error(
+      "We could not resolve workspaces - check you are running this command from the correct directory"
+    );
+  }
+
+  const config = await read(cwd, workspaces);
+
   if (input.length < 1) {
-    await add({ cwd });
+    await add(cwd, config);
   } else if (input.length > 1) {
     logger.error(
       "Too many arguments passed to changesets - we only accept the command name as an argument"
     );
   } else {
     const {
       commit,
-      updateChangelog,
+      changelog,
       skipCI,
-      public: isPublic,
+      access,
       sinceMaster,
       verbose,
       output,
       otp
-    } = flags;
+    }: CliOptions = flags;
 
     // Command line options need to be undefined, otherwise their
     // default value overrides the user's provided config in their
     // config file. For this reason, we only assign them to this
     // object as and when they exist.
-    const config = { cwd };
+
     try {
       switch (input[0]) {
         case "init": {
-          await init({ cwd });
+          await init(cwd);
           return;
         }
         case "add": {
           if (commit !== undefined) {
             config.commit = commit;
           }
-          await add(config);
+          await add(cwd, config);
           return;
         }
-        case "bump": {
+        case "version": {
           // We only assign them to this
           // object as and when they exist.
-          if (updateChangelog !== undefined) {
-            config.updateChangelog = updateChangelog;
-          }
-          if (skipCI !== undefined) {
-            config.skipCI = skipCI;
+          if (changelog !== undefined) {
+            config.changelog = [changelog, null];
           }
           if (commit !== undefined) {
             config.commit = commit;
           }
-          await bump(config);
+          if (skipCI !== undefined) {
+            config.commit = "skipCI";
+          }
+
+          await version(config);
           return;
         }
-        case "release": {
-          if (isPublic !== undefined) {
+        case "publish": {
+          if (access !== undefined) {
             // This exists as
-            config.public = isPublic;
+            config.access = access;
           }
           config.otp = otp;
-          await release(config);
+          await publish(config);
           return;
         }
+        case "bump": {
+          logger.error(
+            'In version 2 of changesets, "bump" has been renamed to "version" - see our changelog for an explanation'
+          );
+          logger.error(
+            "To fix this, use `changeset version` instead, and update any scripts that use changesets"
+          );
+          throw new Error("old command used");
```

Comment:
So that users don't see a stack trace of that error:
```suggestion
          throw new ExitError(1);
```